Tools/Plotfile/fvolumesum.cpp

```diff
@@ -0,0 +1,200 @@
+                                            vol *= dx[idim];
+                                        }
+                                    } else if (coord == 1) {
+                                        // axisymmetric V = pi (r_r**2 - r_l**2) * dz
+                                        //                = pi dr * dz * (r_r + r_l)
+                                        //                = 2 pi r dr dz
+                                        vol = 2 * M_PI * p[0] * dx[0] * dx[1];
+                                    } else if (coord == 2) {
+                                        // 1-d spherical V = 4/3 pi (r_r**3 - r_l**3)
+                                        Real r_r = problo[0]+static_cast<Real>(i+1)*dx[0];
+                                        Real r_l = problo[0]+static_cast<Real>(i)*dx[0];
+                                        vol = (4.0_rt/3.0_rt) * M_PI * dx[0] * (r_r*r_r + r_l*r_r + r_l*r_l);
+                                    }
+
+                                    lsum += fab(i,j,k) * vol;
+                                }
+                            }
+                        }
+                    }
+                }
+            }
+        } else {
+            const MultiFab& mf = pf.get(ilev, var_name);
+            for (MFIter mfi(mf); mfi.isValid(); ++mfi) {
+                const Box& bx = mfi.validbox();
+                if (bx.ok()) {
+                    const auto& fab = mf.array(mfi);
+                    const auto lo = amrex::lbound(bx);
+                    const auto hi = amrex::ubound(bx);
+                    for (int k = lo.z; k <= hi.z; ++k) {
+                        for (int j = lo.y; j <= hi.y; ++j) {
+                            for (int i = lo.x; i <= hi.x; ++i) {
+                                Array<Real,AMREX_SPACEDIM> p
+                                    = {AMREX_D_DECL(problo[0]+static_cast<Real>(i+0.5)*dx[0],
+                                                    problo[1]+static_cast<Real>(j+0.5)*dx[1],
+                                                    problo[2]+static_cast<Real>(k+0.5)*dx[2])};
+
+                                    // compute the volume
+                                    Real vol;
+                                    if (coord == 0) {
+                                        // Cartesian
+                                        for (int idim = 0; idim < AMREX_SPACEDIM; ++idim) {
+                                            vol *= dx[idim];
+                                        }
+                                    } else if (coord == 1) {
+                                        // axisymmetric V = pi (r_r**2 - r_l**2) * dz
+                                        //                = pi dr * dz * (r_r + r_l)
+                                        //                = 2 pi r dr dz
+                                        vol = 2 * M_PI * p[0] * dx[0] * dx[1];
+                                    }
```

Comment:
There is no `coord == 2` here.  But it is supported for `lev < fine_level`.

Reply:
good catch.  Fixed.